package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
Seems like you tried running a command for `npm` in the root of the repo. You can remove this file, as it's unnecessary here.

Reply:
Yea it was late when I was doing this lol. My bad.